xdmod/README.md

```diff
@@ -62,39 +66,188 @@ The following information is needed by Open XDMoD:
+
+Reference: [Configuration Guide](https://open.xdmod.org/configuration.html)
+
+The following asciinema recordings are how an administrator would do perform these actions:
+
+General Setup:
+[![asciicast](https://asciinema.org/a/349236.svg)](https://asciinema.org/a/349236)
+
+Database Setup:
+[![asciicast](https://asciinema.org/a/349237.svg)](https://asciinema.org/349237)
+
+Organization Setup:
+[![asciicast](https://asciinema.org/a/349238.svg)](https://asciinema.org/349238)
+
+Resource Setup:
+[![asciicast](https://asciinema.org/a/349240.svg)](https://asciinema.org/349240)
+
+#### What about a Heterogeneous cluster?
+
+`xdmod-setup` is for the general use case. For some configuration you will need to edit
+Open XDMoD's [Configuration](https://open.xdmod.org/configuration.html#location-of-configuration-files) files.
+
+Resource Specs Modification:
+[![asciicast](https://asciinema.org/a/349249.svg)](https://asciinema.org/349249)
+
+
+#### Hierarchy
+
+Open XDMoD supports a three level hierarchy.
+In this tutorial we use a University Type Hierarchy.
+
+Decanal Unit -> Department -> PI Group
+
+Reference: [Hierarchy Guide](https://open.xdmod.org/hierarchy.html)
+
+#### User / PI Names
+Open XDMoD has the ability to import a full names.
+
+This has not been automated for this tutorial.
+
+```csv
+cgray,Carl,Gray
+sfoster,Stephanie,Foster
+csimmons,Charles,Simmons
+astewart,Andrea,Stewart
+hpcadmin,HPC, Administrators
+```
+
+Reference: [User/PI Names Guide](https://open.xdmod.org/user-names.html)
 
 
 ## Open XDMoD Job Performance
 The Job Performance module is optional, but highly recommended.
-The Job Performance
-TODO: PCP Configuration (mention alternatives, TACC,...)
-https://github.com/ubccr/hpc-toolset-tutorial/blob/master/slurm/pmlogger-supremm.config#L56-L59
 
-Done: Names.csv
+![Job Performance Dataflow](./tutorial-screenshots/admin-job-performance-dataflow.png)
+
+### Job Performance Configuration
 
-TODO: FOS
+This tutorial uses the [Job Performance](https://supremm.xdmod.org) module for Open XDMoD. The Job Performance module displays job performance metric based on data collected on the HPC compute nodes.
+
+The example in this tutorial uses [Performance Co-Pilot (PCP)](https://pcp.io)  as the performance data collection software. This must be [installed](https://github.com/ubccr/hpc-toolset-tutorial/blob/master/slurm/install.sh#L80-L87) and configured on the compute nodes.
+
+This tutorial uses a cut-down list of PCP metrics from the recommended metrics for a production HPC system.
+This shorter list is suitable for running inside the docker demo. On a
+real HPC system the data collection should be setup following the
+[PCP Data collection](https://supremm.xdmod.org/supremm-compute-pcp.html#configuration-templates) guide
+
+The file used in this demo can be viewed here: https://github.com/ubccr/hpc-toolset-tutorial/blob/master/slurm/pmlogger-supremm.config#L56-L59
+
```

Comment:
VERY IMPORTANT - Don't start the configuration of the Job Performance module until there is job data ingested into Open XDMoD. The Job performance setup relies on the accounting data from the Jobs realm in Open XDMoD.